src/pages/orphaned.js

```diff
@@ -0,0 +1,369 @@
+import React, { useRef, useState } from 'react';
+import { StaticQuery, graphql } from 'gatsby';
+import PropTypes from 'prop-types';
+import { config } from '@fortawesome/fontawesome-svg-core';
+import Breadcrumb from '../components/breadcrumb/breadcrumb';
+import { Link } from 'gatsby';
+import { faExclamationTriangle } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { ApplicationInsights } from '@microsoft/applicationinsights-web';
+import { faGithub } from '@fortawesome/free-brands-svg-icons';
+import {
+  faArrowCircleRight,
+  faPencilAlt,
+} from '@fortawesome/free-solid-svg-icons';
```

Comment:
Consolidate fontawesome imports
```suggestion
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { ApplicationInsights } from '@microsoft/applicationinsights-web';
import { faGithub } from '@fortawesome/free-brands-svg-icons';
import {
  faArrowCircleRight,
  faPencilAlt,
  faExclamationTriangle
} from '@fortawesome/free-solid-svg-icons';
```